test/dynamo/test_misc.py

```diff
@@ -4698,6 +4699,40 @@ def test_remove_dead_code_with_exn_table_entries(self):
         self.assertEquals(tab[0].end, 4)
         self.assertEquals(tab[0].target, 6)
 
+    def test_unhandled_exception_in_dynamo(self):
+        # need to run in a separate process to properly test an unhandled
+        # exception raised in dynamo optimized code
+        code = """\
+import torch
+import torch._dynamo
+def f(a):
+    a += 1
+    raise RuntimeError("smoge")
+    return a
+
+opt_fn = torch._dynamo.optimize("eager")(f)
+opt_fn(torch.ones(2))
+"""
+        proc = subprocess.run(
+            ["python3", "-c", code],
```

Comment:
Please always use `sys.executable` as `python3` can not be the version of Python this test was executed with. Also, why not use `multiprocessing`
```suggestion
            [sys.executable, "-c", code],
```